gems/smithy/lib/smithy/anvil/client/templates/endpoint_parameters.erb

```diff
@@ -4,7 +4,14 @@
 
 module <%= namespace %>
   # Endpoint parameters used to resolve endpoints per request.
-  <%= documentation %>
+<% parameters.each do |param| -%>
+  # @!attribute <%= param.name %>
+<% if param.documentation -%>
+  #   <%= param.documentation %>
```

Comment:
What happens when documentations have newline characters in them? Most of the AWS docs have this. General thought for all of documentation rendering. We probably want to always iterate documentation (this method should split by "\n" and return an array?) and if ends up empty, the "if" case is handled, and if it is a single or multiline string, it's handled.

Reply:
I think all existing Endpoint parameter docs are single line, but I dont *think* theres smithy level validation for that, so I'll add handling.